administration/src/mui-modules/application/forms/StepCardTypeForm.tsx

```diff
@@ -105,15 +128,23 @@ const StepCardTypeForm: Form<State, Options, ValidatedInput, AdditionalProps> =
         <CustomDivider />
         <Typography>
           Die Ehrenamtskarte ist als physische Karte und als digitale Version für Ihr Smartphone oder Tablet erhältlich.
-          Hier können Sie wählen, ob Sie neben der physischen auch kostenfrei die digitale Ehrenamtskarte beantragen
-          möchten.
+          Hier können Sie auswählen welchen Kartentyp Sie beantragen möchten.
         </Typography>
         <SubForms.wantsDigitalCard.Component
           state={state.wantsDigitalCard}
           setState={useUpdateStateCallback(setState, 'wantsDigitalCard')}
-          label='Ich beantrage neben der physischen auch die digitale Ehrenamtskarte.'
+          label='Ich beantrage eine digitale Ehrenamtskarte.'
           options={wantsDigitalCardOptions}
         />
+        <SubForms.wantsPhysicalCard.Component
+          state={state.wantsPhysicalCard}
+          setState={useUpdateStateCallback(setState, 'wantsPhysicalCard')}
+          label='Ich beantrage eine physische Ehrenamtskarte.'
+          options={wantsPhysicalCardOptions}
+        />
+        {isInvalid && validationResult.code === CARD_TYPE_ERROR && (
+          <CardTypeAlert severity='error'>{validationResult.message}</CardTypeAlert>
+        )}
```

Comment:
Is it necessary to introduce `validationResult.code` just for this?
Could we not simply show the error only if there is a message?

```suggestion
        {isInvalid && validationResult.message && (
          <CardTypeAlert severity='error'>{validationResult.message}</CardTypeAlert>
        )}
```

Reply:
sure we can. i just thought this is cleaner